udf-compiler/src/main/scala/com/nvidia/spark/udf/CFG.scala

```diff
@@ -171,72 +194,88 @@ object CFG {
       val nextOffset: Int = codeIterator.lookAhead
       val opcode: Int = codeIterator.byteAt(offset)
       // here we are looking for branching instructions
-      opcode match {
-        case Opcode.IF_ICMPEQ | Opcode.IF_ICMPNE | Opcode.IF_ICMPLT |
-             Opcode.IF_ICMPGE | Opcode.IF_ICMPGT | Opcode.IF_ICMPLE |
-             Opcode.IFEQ | Opcode.IFNE | Opcode.IFLT | Opcode.IFGE |
-             Opcode.IFGT | Opcode.IFLE | Opcode.IFNULL | Opcode.IFNONNULL =>
-          // an if statement has two other offsets, false and true branches.
+      if ((opcode == Opcode.ARETURN ||
```

Comment:
This could be just added to the `opcode match {` below as a `case`?
That would minimize diff in the PR